modules/nodes-custom-scheduler-deploying.adoc

```diff
@@ -71,7 +99,7 @@ spec:
         - /usr/local/bin/kube-scheduler
         - --address=0.0.0.0
         - --leader-elect=false
```

Comment:
This line can be removed and moved to the kubescheduler.config.k8s.io/v1beta2 as:
```
    apiVersion: kubescheduler.config.k8s.io/v1beta2
    kind: KubeSchedulerConfiguration
    profiles:
      - schedulerName: custom-scheduler
    leaderElection:
      leaderElect: false
```